### PR TITLE
Add information about SAP PostgreSQL hyperscaler selection and config…

### PR DESCRIPTION
**Pull request by MarkvanMents**
…uration

### CODE REVIEW

File: content/developerportal/deploy/sap-cloud-platform.md [hunk 1]
```diff
@@ -475,14 +475,30 @@ Mendix needs access to a relational database backend and can run using different
 
 ### 8.1 Running Mendix on PostgreSQL
 
-{{% alert type="warning" %}}
-SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that SAP accounts created recently will not be able to directly bind PostgreSQL databases and you will have to use [SAP HANA](#sap-hana) or [AWS RDS PostgreSQL](#aws-rds) databases.
```
Comment (Nidhi251289) [inline content/developerportal/deploy/sap-cloud-platform.md:479]: The above sentence still makes sense. Kindly don't remove this line

File: content/developerportal/deploy/sap-cloud-platform.md [hunk 2]
```diff
@@ -475,14 +475,30 @@ Mendix needs access to a relational database backend and can run using different
...
-{{% /alert %}}
+When you create your environment on SAP Cloud Platform, you can select a PostgreSQL database. This uses the **PostgreSQL on SAP Cloud Platform** database.
+
```
Comment (Nidhi251289) [inline content/developerportal/deploy/sap-cloud-platform.md:479]: image
 We still have postgres, AWS RDS postgres and postgressql hyperscaler databases.

**Review by Nidhi251289 — COMMENTED** [review]
Hi Mark,

I have provided my review comments. Can you also mention that in order to use Azure storage file, you need to use Mendix 8.7.0 and above

File: content/developerportal/deploy/sap-cloud-platform.md [hunk 3]
```diff
@@ -475,14 +475,30 @@ Mendix needs access to a relational database backend and can run using different
 
 ### 8.1 Running Mendix on PostgreSQL
 
-{{% alert type="warning" %}}
-SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that SAP accounts created recently will not be able to directly bind PostgreSQL databases and you will have to use [SAP HANA](#sap-hana) or [AWS RDS PostgreSQL](#aws-rds) databases.
-{{% /alert %}}
+When you create your environment on SAP Cloud Platform, you can select a PostgreSQL database. This uses the **PostgreSQL on SAP Cloud Platform** database.
```
Comment (MarkvanMents) [inline content/developerportal/deploy/sap-cloud-platform.md:478]:
```suggestion
{{% alert type="warning" %}}
SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that SAP accounts created recently will not be able to directly bind PostgreSQL databases and you will have to use [SAP HANA](#sap-hana) or [AWS RDS PostgreSQL](#aws-rds) databases.
{{% /alert %}}

When you create your environment on SAP Cloud Platform, you can still select a PostgreSQL database. You can use either the **PostgreSQL on SAP Cloud Platform** database or the **AWS RDS PostgreSQL** database.
```

File: content/developerportal/deploy/sap-cloud-platform.md (hunk 2 above)
Comment (MarkvanMents) [reply content/developerportal/deploy/sap-cloud-platform.md:479]: Changed above.

**Review by MarkvanMents — COMMENTED** [review]
Hi Nidhi - 
Have added the information back in again. Will also need to make another change to ensure the structure of the document stays good.

File: content/developerportal/deploy/sap-cloud-platform.md (hunk 1 above)
Comment (MarkvanMents) [reply content/developerportal/deploy/sap-cloud-platform.md:479]: Added this back in again.

**Comment by MarkvanMents** [comment]
Release Approval over Slack.

**Pull request merged**